resources/translations/en.edn

```diff
@@ -7,6 +7,8 @@
             :add-member "Add member"
             :approve "Approve"
             :approve-reject "Approve or reject"
+            :assign-external-id "Assign external id"
+            :assign-external-id-info "External id"
```

Comment:
Is this info supposed to be some longer description?

Reply:
Or just `:external-id`?

Reply:
We'll probably have some longer text later but my head was empty